### DISCUSSION TITLE
Support draft pull requests on gitlab

### DISCUSSION THREAD

**Discussion by elondres-mim** [magit/forge#652] [Ideas]
Similar to how draft PRs are supported for GitHub, it would be neat if forge supported draft MRs on GitLab.

I got a bit confused in pullreq-mode because the Draft line appears for GitLab MRs, but editing it doesn't do anything on the remote. Of course the workaround for now, because of how GitLab works, is just to edit the title to remove 'Draft:'.

**Comment by tarsius** [comment]
> I got a bit confused in pullreq-mode because the Draft line appears for GitLab MRs, but editing it doesn't do anything on the remote.

That works now.

However, when first creating the merge-request, you have to manually prefix the title with `Draft: `.

That's good enough. I do not plan to invest much time improving Gitlab support.
